tuning/autotune.py

```diff
@@ -607,106 +724,111 @@ def compile_candidates(
         level=logging.WARNING,
     )
 
-    return compiled_files, compiled_dir
+    return compiled_candidates
+
+
+def parse_dispatch_benchmark_results(
+    path_config: PathConfig,
+    benchmark_results: list[TaskResult],
+    candidate_trackers: CandidateTracker,
+) -> list[tuple[str, str, str]]:
```

Comment:
Can we make this produce some `class` instead of `tuple[str, str, str]`. Just looking at this, it's hard to tell what the result contains.